<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DReportAllViolationAVP</class>
 <widget class="QDialog" name="DReportAllViolationAVP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчеты общей статистики АВП с нарушениями</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Фильтры</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBoxViolation">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>По нарушениям</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QComboBox" name="comboBoxViolation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxDate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>По дате проведения экспертного исследования</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="3">
          <widget class="QDateEdit" name="dateEditDateEnd">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2020</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDateEdit" name="dateEditDateBegin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2020</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>С:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>По:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonReview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Обновить</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/icons/button_reload.png</normaloff>:/icons/icons/button_reload.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Дата отчета</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата исследования</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Наименование АВС</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Наименование АВП</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Тип нарушения</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Описание нарушения законодательства</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Принятые меры</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время нарушения</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ссылка на АВП</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Комментарий РКН</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус отправки</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>103</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Сформировать отчет</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/edit.png</normaloff>:/icons/icons/edit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonExit">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>DReportAllViolationAVP</receiver>
   <slot>slotExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1182</x>
     <y>841</y>
    </hint>
    <hint type="destinationlabel">
     <x>1261</x>
     <y>743</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DReportAllViolationAVP</receiver>
   <slot>slotMakeReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1088</x>
     <y>835</y>
    </hint>
    <hint type="destinationlabel">
     <x>1261</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonReview</sender>
   <signal>clicked()</signal>
   <receiver>DReportAllViolationAVP</receiver>
   <slot>slotReview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1207</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>1263</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotExit()</slot>
  <slot>slotReview()</slot>
  <slot>slotMakeReport()</slot>
 </slots>
</ui>
